### PR TITLE
apps/examples: Add missing Kconfig_ENTRY files in sub-directories

### PR DESCRIPTION
**Pull request by jeongarmy**
- apps/examples: Add missing Kconfig_ENTRY files in sub-directories.

### CODE REVIEW

File: apps/examples/board_specific_demo/Kconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#
```
Comment (jeongchanKim) [inline apps/examples/board_specific_demo/Kconfig:1]: Only some files have the under descriptions, some are not.
Could you add if needed?
```
# For a description of the syntax of this configuration file,
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
#
```

File: apps/examples/security_test/Kconfig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+menu "Security Test"
+source "/root/tizenrt/apps/examples/security_test/hal/Kconfig"
+source "/root/tizenrt/apps/examples/security_test/mbedtls_self/Kconfig"
+source "/root/tizenrt/apps/examples/security_test/seclink/Kconfig"
+source "/root/tizenrt/apps/examples/security_test/security_api/Kconfig"
```
Comment (jeongchanKim) [inline apps/examples/security_test/Kconfig:10]: Only `security_api` doesn't have `Kconfig_ENTRY`.
Could you check this is okay or not?

**Review by sunghan-chang — COMMENTED** [review]
I disagree this way because this makes different policy for Kconfig including. User could confuse whether including Kconfig is necessary or not.

**Review by sunghan-chang — CHANGES_REQUESTED** [review]
Please find another way to include them automatically like other 1 depth Kconfig of the example.

**Review by amogh-samsung — COMMENTED** [review]
I think "performance" and "wifi_manager" subfolders in examples still need Kconfig dependencies resolved since they have sub folders inside them which have Kconfigs

File: apps/examples/smartfs/buffer_optimization/Kconfig_ENTRY [hunk 3]
```diff
@@ -0,0 +1,3 @@
+config ENTRY_SMARTFS_BUFFER_OPTIMIZATION
```
Comment (jeongchanKim) [inline apps/examples/smartfs/buffer_optimization/Kconfig_ENTRY:1]: How about changing `ENTRY_SMARTFS_BUFFER_OPTIMIZATION` to `ENTRY_SMARTFS_BUF_OPTIMIZATION`?
Another configs are matched with the original config name, but this config is not.

File: apps/examples/board_specific_demo/Kconfig_ENTRY [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+source "/root/tizenrt/apps/examples/board_specific_demo/artik_demo/Kconfig_ENTRY"
+source "/root/tizenrt/apps/examples/board_specific_demo/cy4390x_demo/Kconfig_ENTRY"
+source "/root/tizenrt/apps/examples/board_specific_demo/esp32_wifi_demo/Kconfig_ENTRY"
```
Comment (sunghan-chang) [inline apps/examples/board_specific_demo/Kconfig_ENTRY:8]: Same as previous comment. I disagree this manual including.

File: apps/examples/libcoap/server/Kconfig_ENTRY [hunk 5]
```diff
@@ -0,0 +1,3 @@
+config ENTRY_LIBCOAP_SERVER_TEST
+	bool "\"libcoap Server Test\" example"
```
Comment (sunghan-chang) [inline apps/examples/libcoap/server/Kconfig_ENTRY:2]: How about removing `"  "`?
```
"libcoap Server Test"
```
looks enough.

File: apps/examples/libcoap/client/Kconfig_ENTRY [hunk 6]
```diff
@@ -0,0 +1,3 @@
+config ENTRY_LIBCOAP_CLIENT_TEST
+	bool "\"libcoap Client Test\" example"
```
Comment (sunghan-chang) [inline apps/examples/libcoap/client/Kconfig_ENTRY:2]: same as previous

File: apps/examples/smartfs/smart/Kconfig_ENTRY [hunk 7]
```diff
@@ -0,0 +1,3 @@
+config ENTRY_SMART
+	bool "SMART file system example"
```
Comment (kishore-sn) [inline apps/examples/smartfs/smart/Kconfig_ENTRY:2]: Can you pls make "file system" ->"filesystem"?

File: apps/examples/smartfs/buffer_optimization/Kconfig_ENTRY [hunk 8]
```diff
@@ -0,0 +1,3 @@
+config ENTRY_SMARTFS_BUF_OPTIMIZATION
+	bool "SmartFs Buffer Optimization Example"
```
Comment (kishore-sn) [inline apps/examples/smartfs/buffer_optimization/Kconfig_ENTRY:2]: Example -> example

Also, can we make a uniform standard about where capital letters must be used?

Comment (sunghan-chang) [reply apps/examples/smartfs/buffer_optimization/Kconfig_ENTRY]: I think we need to check entire examples for what you mentioned. Let's merge as it is and then check all. How do you think?

Comment (sunghan-chang) [reply apps/examples/smartfs/buffer_optimization/Kconfig_ENTRY]: Thank you.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**